en/components/navbar.md

```diff
@@ -97,27 +97,11 @@ Next, we need to update our template with an icon for each of the options we wan
 
 <div class="divider"></div>
 
-### API Summary
+### API References
 
-In this article we show a few scenarios where the navbar component may come in handy. The APIs, we used to achieve them, are listed below.
+In this article we show a few scenarios where the navbar component may come in handy. The APIs, we used to achieve them, are listed in the links below.
 
-####Inputs
-The following inputs are available on the **igx-navbar** component:
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `actionButtonIcon` | String | Provide the name of the material icon to be used in the action button. |
-| `isActionButtonVisible` | Boolean | Whether the action button should be displayed. |
-| `title` | String | Provide the title the navbar should display. |
-
-<div class="divider"></div>
-
-####Outputs
-The following outputs are available on the **igx-navbar** component:
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `onAction` | EventEmitter | Emits an event when the action is executed. |
-
-<div class="divider"></div>
+[`IgxNavbarComponent`]({environment:angularApiUrl}/classes/igxnavbarcomponent.html)
```

Comment:
Provide Sassdoc reference for the [igx-navbar-theme](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/#function-igx-navbar-theme).

Also in the topic is mentioned the IgxIcon, for more information about this component you could provide Typedoc and Sassdoc references to its [API ](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxiconcomponent.html) and [theme](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/#function-igx-icon-theme)